airbyte-server/src/test/java/io/airbyte/server/migration/RunMigrationTest.java

```diff
@@ -96,6 +97,7 @@ public void cleanup() throws IOException {
 
   @SuppressWarnings("UnstableApiUsage")
   @Test
+  @Disabled
```

Comment:
@tuliren this also fails.

Reason:
1) StubDB is set up to use the latest schema from airbyte_db module.
2) Migrations fails since the migrations are from the old system and don't have the latest schemas.

Reply:
What if we 'freeze' the db schema in a different file so the migration tests pass for now?

Reply:
This is also making me wonder if we missed database archive migration.

I understand Flyway will handle database schema migration - is it also set up to handle migrating the database archive?

Reply:
No, Flyway cannot handle archive migration. It only runs in db. Once the table structure changes, the file data cannot be loaded into db.

I have a plan to handle this and created an issue to track it.

Reply:
> What if we 'freeze' the db schema in a different file so the migration tests pass? Think it's important to keep them until we get rid of this entirely.

Yes, we can pass in the last file based migration schema as the "latest" schema in this kind of unit test.

Reply:
Given all the questions you have raised, I think it is the best for the long term to completely deprecate the file based migrations and write Flyway replacement for them.

Reply:
Issue: https://github.com/airbytehq/airbyte/issues/5902